packages/cashier/src/types/shared/websocket.types.ts

```diff
@@ -1,5 +1,34 @@
+import {
+    CashierInformationRequest,
+    CashierInformationResponse,
+    CryptoConfig,
+    GetAccountSettingsResponse,
+    PaymentAgentTransferRequest,
+    PaymentAgentTransferResponse,
+} from '@deriv/api-types';
+
 export type TServerError = {
     code: string;
     message: string;
     details?: { [key: string]: string };
 };
+
+type TWebSocketCall = {
+    cashier: (
+        action: CashierInformationRequest['cashier'],
+        parameters: Omit<CashierInformationRequest, 'cashier'>
+    ) => Promise<CashierInformationResponse>;
+    paymentAgentTransfer: (
+        props: Omit<PaymentAgentTransferRequest, 'paymentagent_transfer'>
+    ) => Promise<PaymentAgentTransferResponse>;
+    cryptoConfig: () => { crypto_config: CryptoConfig };
+    cryptoWithdraw: (
+        args: Omit<CashierInformationRequest, 'cashier' | 'provider' | 'type'>
+    ) => Promise<CashierInformationResponse>;
+    send: (obj: any) => Promise<any>;
+    storage: {
+        getSettings: () => Promise<GetAccountSettingsResponse>;
+    };
+};
+
+export type TWebSocket = { authorized: TWebSocketCall } & TWebSocketCall;
```

Comment:
@farzin-fs Why use the same type for the `authorized` object and the outer object of the `TWebSocket`?

There are some WS calls that do not belong to `authorized` and vice versa.

Reply:
@hamid-deriv Yes you are correct, I'm just trying to avoid code duplication as this is just a temporary solution for typing socked calls, Later we are going to use your idea that you raised in the channel so we won’t need this file anymore.

Reply:
But I think this will produce ts errors as the properties inside the `authorized` are not the same as properties outside of it. I think it's better to define the properties for outside of the `authorized` property.

Reply:
@hamid-deriv Sure I will update this 🙇🏻

Reply:
@hamid-deriv Updated 🙇🏻